tests/app/admin_Tokens_POST_test.py

```diff
@@ -180,7 +189,61 @@ def test_normal_2(self, client: TestClient, session: Session, shared_contract):
 
         register_bond_list(issuer, bond_token, token_list)
 
-        request_params = self.token_2
+        request_params = self.token_param_2
+        request_params["contract_address"] = bond_token["address"]
+        headers = {'Content-Type': 'application/json'}
+        request_body = json.dumps(request_params)
+        resp = client.post(self.apiurl, headers=headers, json=json.loads(request_body))
+
+        assert resp.status_code == 200
+        assert resp.json()['meta'] == {'code': 200, 'message': 'OK'}
+
+        listing: Listing = session.query(Listing). \
+            filter(Listing.token_address == bond_token["address"]). \
+            first()
+        assert listing.token_address == bond_token["address"]
+        assert listing.is_public == self.token_param_2["is_public"]
+        assert listing.max_holding_quantity is None
+        assert listing.max_sell_amount is None
+        assert listing.owner_address == issuer["account_address"]
+
+        bond: IDXBondToken = session.query(IDXBondToken).first()
+        assert bond.token_address == bond_token["address"]
+        assert bond.owner_address == issuer["account_address"]
+
+        position: IDXPosition = session.query(IDXPosition).first()
+        assert position.token_address == bond_token["address"]
+        assert position.account_address == issuer["account_address"]
+        assert position.balance == 1000000
+
+    # <Normal_3>
+    # Position data has already indexed
+    def test_normal_3(self, client: TestClient, session: Session, shared_contract):
```

Comment:
Added a case where there is a position data about given contract_address and owner_address.